server/graphql/types/src/types/vaccine_course_item.rs

```diff
@@ -3,42 +3,44 @@ use dataloader::DataLoader;
 
 use graphql_core::{loader::ItemLoader, standard_graphql_error::StandardGraphqlError, ContextExt};
 
-use repository::vaccine_course::vaccine_course_item_row::VaccineCourseItemRow;
+use repository::vaccine_course::vaccine_course_item::VaccineCourseItem;
 
 use super::ItemNode;
 
 #[derive(PartialEq, Debug)]
 pub struct VaccineCourseItemNode {
-    pub vaccine_course_item: VaccineCourseItemRow,
+    pub vaccine_course_item: VaccineCourseItem,
 }
 
 #[Object]
 impl VaccineCourseItemNode {
     pub async fn id(&self) -> &str {
-        &self.row().id
+        &self.row().vaccine_course_item.id
     }
 
     pub async fn item(&self, ctx: &Context<'_>) -> Result<ItemNode> {
```

Comment:
If it can be done easily, It would be nice to remove the loader call.
I suspect others kept the loader for the same reason you did, it was the easiest option.

Reply:
Quick I think! Will do now